zulipterminal/themes/zt_dark.py

```diff
@@ -68,6 +68,7 @@
     'task:error'      : (Color.WHITE,               Color.DARK_RED),
     'task:warning'    : (Color.WHITE,               Color.BROWN),
     'ui_code'         : (Color.BLACK,               Color.WHITE),
+    'guest_suffix'    : (Color.WHITE,               Color.BLACK),
```

Comment:
This is one place where a style can be added to a theme, but note there are others - it will need adding in each place.

For example, look for one of the other style names throughout the code.

See previous style-adding commits in the git log for examples - we tend to add a style in all places in a single commit.

Reply:
yes i saw there are a lot of styles i just wanted to show this to know if im on the right track or not, will add it in all the files

Reply:
I have done it in all the files right ?